<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Path</class>
 <widget class="QDialog" name="Path">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Path</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QFrame" name="frameAll">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout">
        <item>
         <widget class="QFrame" name="frameIon">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>200</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>32767</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frameTrek">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>200</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
     <zorder></zorder>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEditInf">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32767</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonRead">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Input FileName</string>
       </property>
       <property name="text">
        <string>In File</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonWrite">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Input FileName</string>
       </property>
       <property name="text">
        <string>Out File</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonWriteMUF">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Input FileName</string>
       </property>
       <property name="text">
        <string>MUF File</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonRun">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Run Calculated</string>
       </property>
       <property name="text">
        <string>Run</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonExit">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Exit</string>
       </property>
       <property name="text">
        <string>Exit</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>pushButtonExit</tabstop>
  <tabstop>pushButtonRead</tabstop>
  <tabstop>pushButtonRun</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonExit</sender>
   <signal>pressed()</signal>
   <receiver>Path</receiver>
   <slot>slotExit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>523</x>
     <y>530</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonRun</sender>
   <signal>pressed()</signal>
   <receiver>Path</receiver>
   <slot>slotRun()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>439</x>
     <y>530</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonRead</sender>
   <signal>pressed()</signal>
   <receiver>Path</receiver>
   <slot>slotRead()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>530</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonWriteMUF</sender>
   <signal>clicked()</signal>
   <receiver>Path</receiver>
   <slot>slotWriteMUF()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>384</x>
     <y>522</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>539</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonWrite</sender>
   <signal>clicked()</signal>
   <receiver>Path</receiver>
   <slot>slotWrite()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>527</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>540</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotWrite()</slot>
  <slot>slotWriteMUF()</slot>
 </slots>
</ui>
